app/models/goods_nomenclatures/nested_set.rb

```diff
@@ -127,6 +127,7 @@ def initialize
           .dedupe_similar
           .with_regulation_dates_query
           .without_excluded_types
+          .without_excluded_series
```

Comment:
Rather than permanently extending the series, I'm wondering whether we should do a migration to delete these measures?

Reply:
Talked about this offline

The plan is to:

1. Turn these off
2. Observe for an arbitrary period to see if we get complaints/people depending on these
3. Remove the data in a subsequent story